search-app/docs/site.md

```diff
@@ -14,12 +14,15 @@ See also the [API Documentation](search_api_docs.html).
 
```

Comment:
Fixed.

Also did a visual check on the other docs with spell-check enabled and found some old typos -- those are also fixed now.